### PR TITLE
Docs setup contribute

### PR DESCRIPTION
**Pull request by lucianoratamero**
**Link to task:** https://labcodes.atlassian.net/browse/DSYS-89
**Staging app:** https://docs-setup-contribute--labstorybook-master.netlify.app/

This PR adds Setup and How to Contribute stories, as well as README files for the storybook and DS.

### CODE REVIEW

**Review by falconedani — APPROVED** [review]
I think it’s ok, but I was wondering if it wouldn’t be better to create a `Theme` story under "base” with the section you wrote to make it easier to find.

Also, is Storybook really needed to just use the design system? Wouldn’t it be a better fit as a section in How to Contribute?

Minor thingy: could you add target blank so that links open in a different tab? It was very annoying for me.

**Comment by lucianoratamero** [comment]
@falconedani about your points:

1- I'm not sure I agree with your point. :thinking: the theming is 100% related to how we would use the DS' code in an outside project, so, as a developer, it makes more sense for me to have it together with the other instructions on how to set it up. but I do want to understand: why did you think it would be better to put it under "Base" instead? Beucase there may be other solutions, depending on the problem.

2- To answer your question: no, we don't need the storybook to use the DS; but we need the DS to use the storybook. Maybe having both set up instructions together in the same page gave you the impression that they are coupled together, though. Was that the problem? 
If so, I don't think the solution would be to put it on How to Contribute; the whole project is based on storybook, and putting the set up instructions inside the How to Contribute could be even more confusing. My suggestion would be for us to create a separate story for the Confetti instructions. Maybe we could even create their own section of the sidebar, so that we could put the set up instructions for Storybook, Confetti and How to Contribute. What do you think?

3- Sure, working on it!

**Comment by lucianoratamero** [comment]
PS: for these cases, in which things are still confusing and changes will be needed, it's better to not approve the PR. Approving it gives us the idea that it's ready to be merged, when it's not the case. It's not a problem to reject a PR, even if the code is alright. :]

File: stories/base/ChangeLog.stories.mdx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { Fragment } from 'react';
 import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
-import { SimpleTag, TogglableTag, RemovableTag, DropdownTag } from "../labsystem/src/Tags";
+import { SimpleTag, TogglableTag, RemovableTag, DropdownTag } from "../../labsystem/src/Tags";
 
-<Meta title="Getting started/Changelog" />
+<Meta title="Base/Changelog" />
```
Comment (falconedani) [inline stories/base/ChangeLog.stories.mdx:5]:
```suggestion
<Meta title="Getting started/Changelog" />
```

I felt it lost in the base menu, please move it back to getting started

File: stories/getting-started/How to Contribute.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    <h1 class="docs-title__text">How to Contribute</h1>
+  </div>
+</div>
+
+<div class="docs-content">
+
+  <div class="columns">
+    <div class="column">
+      <h2>Getting started</h2>
+      <p>If you're not an active developer on the Confetti team, to start contributing, please fork our <a target="_blank" rel="noopener" href="https://github.com/labcodes/labstorybook/">source code on Github</a>. If you are, you'll already have access to the main repo, so forget about forks. Currently, the codebase contains both the Storybook and the Confetti's own source code, so please read our <a target="_blank" rel="noopener" href="/?path=/docs/getting-started-setup--page">Setup docs</a>.</p>
```
Comment (falconedani) [inline stories/getting-started/How to Contribute.stories.mdx:17]:
```suggestion
      <p>If you're not an active developer on the Confetti team, to start contributing, please fork our <a target="_blank" rel="noopener" href="https://github.com/labcodes/labstorybook/">source code on Github</a>. If you are, you'll already have access to the main repo, so forget about forks. Currently, the codebase contains both the <a target="_blank" rel="noopener" href="/?path=/docs/getting-started-setup-storybook--page">Storybook</a> and the <a target="_blank" rel="noopener" href="/?path=/docs/getting-started-setup-confetti--page">Confetti's</a> own source code, so please read the docs.</p>
```

File: stories/getting-started/How to Contribute.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      <h2>Branching model</h2>
+      <p>We currently use Trunk based development for contributions</p>
+      <p><a target="_blank" rel="noopener" href="https://trunkbaseddevelopment.com/5-min-overview/">Trunk based</a> repositories have only one permanent branch: <code>master</code>. From it, we can create <code>feature</code> and <code>hotfix</code> branches.</p>
+      <p>The only main rule is that <code>master</code> should always, <strong>always</strong> stable. For that, be sure to run the tests (details on how to are in the <a target="_blank" rel="noopener" href="/?path=/docs/getting-started-setup--page">Setup docs</a>) before pushing your code and never push with the <code>--no-validate</code> flag, unless specifically being asked to.</p>
```
Comment (falconedani) [inline stories/getting-started/How to Contribute.stories.mdx:21]:
```suggestion
      <p>The only main rule is that <code>master</code> should always, <strong>always</strong> be stable. For that, be sure to run the tests (details on how to are in the <a target="_blank" rel="noopener" href="/?path=/docs/getting-started-setup--page">Setup docs</a>) before pushing your code and never push with the <code>--no-validate</code> flag, unless specifically being asked to.</p>
```

**Review by falconedani — CHANGES_REQUESTED** [review]
Tiny things to adjust

**Comment by lucianoratamero** [comment]
Done @falconedani ! Merging :]

**Pull request merged**